docs/reference/mpremote.rst

```diff
@@ -245,6 +248,20 @@ The full list of supported commands are:
   ``mpremote fs cp :main.py main.py`` copies ``main.py`` from the device back
   to the current directory.
 
+  .. code-block:: bash
+
+      $ mpremote fs cp main.py :main.py + repl
```

Comment:
This example should go below, the bit documenting the `+` character.

I think it would be better to just add a single paragraph for tree and not reorder anything else.